docs/src/pages/components/divider/react.mdx

```diff
@@ -90,7 +90,13 @@ You can customize the appearance of all Dividers in your application with a [The
   </ExampleCode>
 </Example>
 
-### CSS
+### Styling
+
+### Target classes
```

Comment:
Do you think it would be possible to generate this table from the `ComponentClassNames` enum?